src/com/lftechnology/java/training/sanish/application/component/EmployeeHelper.java

```diff
@@ -0,0 +1,417 @@
+        String message =
+                "\n" + Constants.USER_DEPARTMENT_LABEL + employee.getDepartment() + "\n>>" + Constants.CHANGE_DEPARTMENT_CONFORM_MSG;
+        LOGGER.log(Level.INFO, message);
+        String edit = UserInput.getString(inputScanner);
+        if (edit.equals(Constants.YES_OPTION) || edit.equals(Constants.FULL_YES_OPTION)) {
+            int updateRow = 0;
+            String department;
+            String setStatement;
+            Boolean doChange = true;
+            String whereCondition = "employeeId=?";
+            EmployeeDao employeeDao = new EmployeeDao();
+            while (doChange) {
+                message = "\n >>" + Constants.TYPE_DEPARTMENT_LABEL;
+                LOGGER.log(Level.INFO, message);
+                department = UserInput.getAlphabeticWords(inputScanner);
+                DateFormat dateFormat = new SimpleDateFormat(Constants.DB_DATE_FORMAT);
+                Date date = new Date();
+                setStatement = "department=?, modifiedAt=?";
+                updateRow = employeeDao.update(whereCondition, setStatement, department, dateFormat.format(date), employee.getEmployeeId());
+                if (updateRow == 0) {
+                    message = "\n" + Constants.ERROR_MSG_LABEL + Constants.FAIL_UPDATE_DEPARTMENT_MSG;
+                    doChange = conformRetry(inputScanner, message);
+                }
+            }
+        }
+    }
+
+    /**
+     * Change role of user
+     *
+     * @param inputScanner {@link Scanner}
+     * @param employee     {@link Employee}
+     * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+     */
+    public static void changeRole(Scanner inputScanner, Employee employee) {
+        String message = "\n" + Constants.USER_ROLE_LABEL + employee.getRole() + "\n>>" + Constants.CHANGE_ROLE_CONFORM_MSG;
+        LOGGER.log(Level.INFO, message);
+        String edit = UserInput.getString(inputScanner);
+        if (edit.equals(Constants.YES_OPTION) || edit.equals(Constants.FULL_YES_OPTION)) {
+            int updateRow = 0;
+            String role;
+            String setStatement;
+            Boolean doChange = true;
+            String whereCondition = "employeeId=?";
+            EmployeeDao employeeDao = new EmployeeDao();
+            while (doChange) {
+                message = "\n >>" + Constants.TYPE_ROLE_LABEL;
+                LOGGER.log(Level.INFO, message);
+                role = UserInput.getString(inputScanner);
+                if (role.equals(Constants.ADMIN_ROLE) || role.equals(Constants.USER_ROLE)) {
+                    DateFormat dateFormat = new SimpleDateFormat(Constants.DB_DATE_FORMAT);
+                    Date date = new Date();
+                    setStatement = "role=?, modifiedAt=?";
+                    updateRow = employeeDao.update(whereCondition, setStatement, role, dateFormat.format(date), employee.getEmployeeId());
+                    if (updateRow == 0) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS134)